airflow/utils/db.py

```diff
@@ -993,7 +993,7 @@ def decrypt_trigger_kwargs(*, session: Session) -> None:
         # this can happen when we downgrade to an old version before the Trigger table was added
         return
 
-    for trigger in session.query(Trigger):
+    for trigger in session.scalars(select(Trigger.encrypted_kwargs)):
```

Comment:
This is just an improvement or it's a bug fix too?

Reply:
This is the bug fix itself

Reply:
I just checked the PR description, it makes sense. However I don't why it didn't fail when I tested it with breeze.